v2/rust/src/setup.rs

```diff
@@ -72,3 +77,63 @@ fn remove_files_atomic<'a>(
 
     Ok(())
 }
+
+fn adjust_user_permissions(config: &Config) {
+    for (suite_name, suite_config) in config.suites() {
+        let _ = adjust_user_permissions_for_suite(&config.working_directory, suite_config)
+        .context(format!("Adjusting user permissions for suite {suite_name} failed, which might impair suite execution"))
+        .map_err(log_and_return_error);
+    }
+}
+
+fn adjust_user_permissions_for_suite(
+    working_directory: &Utf8Path,
+    suite_config: &SuiteConfig,
+) -> Result<()> {
+    let user_name = match &suite_config.session_config {
+        SessionConfig::SpecificUser(user_session_config) => &user_session_config.user_name,
+        _ => return Ok(()),
+    };
+
+    debug!("Granting {user_name} full access to {working_directory}");
+    run_icacls_command(vec![
+        working_directory.as_str(),
+        "/grant",
+        &format!("{user_name}:(OI)(CI)F"),
```

Comment:
I don't understand what this list means?

Reply:
Are containers just directories and objects just files?

Reply:
I think so, but I couldn't find any official documentation on this. Windows ...